### PR TITLE
web screenshot

### PR DESCRIPTION
**Pull request by WhiteBoardDev**

### CODE REVIEW

File: hubot-web-screenshot/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# hubot-web-screenshot
+
+Takes a picture of a website and shows it to you. Currently works with slack.
+
+# Setup
+
+Add hubot app to slack and create an API key. Set the API key as an environment variable like so
+
+```
+HUBOT_SLACK_TOKEN=xoxb-xxcjqneigb3415-452130989397-314i13jtn24jgn4534
```
Comment (jonatan-ivanov) [inline hubot-web-screenshot/README.md:10]: Ooops? :)

File: hubot-web-screenshot/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  "version": "1.0.0",
+  "description": "Takes a screenshot of a website and posts the resulting image. Supports various authentication methods.",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha",
+    "debug-test": "mocha --debug-brk --inspect"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rheobot-team/rheobot.git"
```
Comment (jonatan-ivanov) [inline hubot-web-screenshot/package.json:12]: git+?

**Review by jonatan-ivanov — APPROVED** [review]

File: hubot-web-screenshot/README.md (hunk 1 above)
Comment (WhiteBoardDev) [reply hubot-web-screenshot/README.md:10]: The token isn't actually real. It looks like the same format as a slack token but i made it up by dragging my face across the keyboard a few times.

File: hubot-web-screenshot/package.json (hunk 2 above)
Comment (WhiteBoardDev) [reply hubot-web-screenshot/package.json:12]: that was auto geneated by npm init. I guess that must be the convention if you support clone via SSH and also HTTPS?

**Pull request merged**